openstudiocore/src/model/EnergyManagementSystemProgram.cpp

```diff
@@ -114,7 +114,8 @@ namespace detail {
   bool EnergyManagementSystemProgram_Impl::setBody(const std::string& body) {
     //set body of program to input string
     bool result = false;
-    bool comment_result = false;
+    // TODO: JM 2019-01-03: This is unused
+    [[maybe_unused]] bool comment_result = false;
```

Comment:
Hey @brianlball, would `comment_result` be used in the future for EMS program/subroutine? I'd like to avoid using `[[maybe_unused]]` flags as possible